tests/test_parallel_loading_metrics.py

```
@@ -0,0 +1,128 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)